shub/image/utils.py

```diff
@@ -157,6 +157,9 @@ def get_credentials(username=None, password=None, insecure=False,
     without 'account' query parameter which breaks login.
     """
     if insecure:
+        if username or password:
+            raise click.BadParameter(
```

Comment:
Could you please replace it with [BadParameterException](https://github.com/scrapinghub/shub/blob/master/shub/exceptions.py#L57)? It's nearly the same as original `BadParameter`, but I would prefer to be consistent with the other code.

Reply:
Sure!

Reply:
One thing, the `BadParameterException` lives in `shub.exceptions`, not in `click`

Reply:
I'm checking right now and in the rest of the function it is used `BarParameter`. For consistency do you refer to other code in shub? Should the lines below 160 too? https://github.com/scrapinghub/shub/blob/6b71fb2ea378e8c659629219617739c9eda286c3/shub/image/utils.py#L168

Reply:
Oh, you're right indeed, I missed that. 
> For consistency do you refer to other code in shub?

Yeah, exactly. Could you please replace the other `click.BadParameter` in the function with `BadParameterException` from `shub.exceptions`?